frontend/src/components/tabs/RuntimeNodeDetailsV2.tsx

```diff
@@ -35,16 +35,22 @@ import {
 } from 'src/mlmd/MlmdUtils';
 import { NodeMlmdInfo } from 'src/pages/RunDetailsV2';
 import { ArtifactType, Execution } from 'src/third_party/mlmd';
-import ArtifactPreview from '../ArtifactPreview';
-import Banner from '../Banner';
-import DetailsTable from '../DetailsTable';
-import { FlowElementDataBase } from '../graph/Constants';
-import LogViewer from '../LogViewer';
-import { getResourceStateText, ResourceType } from '../ResourceInfo';
-import { MetricsVisualizations } from '../viewers/MetricsVisualizations';
+import ArtifactPreview from 'src/components/ArtifactPreview';
+import Banner from 'src/components/Banner';
+import DetailsTable from 'src/components/DetailsTable';
+import { FlowElementDataBase } from 'src/components/graph/Constants';
+import LogViewer from 'src/components/LogViewer';
+import { getResourceStateText, ResourceType } from 'src/components/ResourceInfo';
+import { MetricsVisualizations } from 'src/components/viewers/MetricsVisualizations';
 import { ArtifactTitle } from './ArtifactTitle';
 import InputOutputTab, { getArtifactParamList } from './InputOutputTab';
 
+export enum LogsInfoKeys {
```

Comment:
Let's not use enum in this case. Directly define the 3 string constants should be sufficient (You can define a shared prefix to declare that they are for the same object, like `LOGS_INFO_`.). If you want to wrap them in a category, you can extract them to a file or define a function that returns const.